### PR TITLE
IBX-4927: Changing custom tag configuration breaks existing Rich Text fields that include this tag

### PR DESCRIPTION
**Pull request by GrabowskiM**
| Question           | Answer
| ------------------ | ------------------
| **JIRA issue**     | https://issues.ibexa.co/browse/IBX-4927
| **Bug/Improvement**| yes
| **New feature**    | no
| **Target version** | 4.3
| **BC breaks**      | no
| **Tests pass**     | yes
| **Doc needed**     | no

**TODO**:
- [x Implement feature / fix a bug.
- [ ] Implement tests.
- [x] Fix new code according to Coding Standards (`$ composer fix-cs`).
- [x] Ask for Code Review.

### CODE REVIEW

File: src/bundle/Resources/public/js/CKEditor/core/base-ckeditor.js [hunk 1]
```diff
@@ -207,13 +207,20 @@ const VIEWPORT_TOP_OFFSET = 102;
...
                     const textarea = container.closest('.ibexa-data-source').querySelector('textarea');
 
                     textarea.value = this.xhtmlify(data).replace(this.xhtmlNamespace, this.ezNamespace);
                     textarea.dispatchEvent(new Event('input'));
+                };
+
+                updateInput(initialData);
+
+                this.editor.model.document.on('change:data', () => {
+                    const data = this.getData();
```
Comment (Gengar-i) [inline src/bundle/Resources/public/js/CKEditor/core/base-ckeditor.js:221]: Is this.editor.getData(); synchronous operation?

**Review by lucasOsti — APPROVED** [review]

File: src/bundle/Resources/public/js/CKEditor/core/base-ckeditor.js (hunk 1 above)
Comment (GrabowskiM) [reply src/bundle/Resources/public/js/CKEditor/core/base-ckeditor.js:221]: Yes. Also that's exact line from https://github.com/ibexa/product-catalog/pull/913 , only indentation was changed :)

**Comment by tomaszszopinski** [comment]
@GrabowskiM After applying patch im getting no error after editing custom tag (which has deleted style section) Is this the intention of this pr?

**Review by tomaszszopinski — APPROVED** [review]
QA approved on IbexaDXP 4.4 commerce.

**Pull request merged**